core/src/main/scala/org/apache/spark/eventhubs/client/package.scala

```diff
@@ -0,0 +1,7 @@
+package org.apache.spark.eventhubs
+
+package object client {
+  implicit def toJavaFunction[A, B](f: Function1[A, B]) = new java.util.function.Function[A, B] {

```

Comment:
Do we need this? I don't see a usage of the API.

Reply:
removed